src/site/includes/debug.drupal.liquid

```diff
@@ -1,5 +1,7 @@
-{% if isPreview or buildtype != 'vagovprod' and omitdebug = false %}
-  {% assign showDebug = true %}
+{% if isPreview or buildtype != 'vagovprod' %}
+  {% if omitdebug = false %}
+    {% assign showDebug = true %}
+  {% endif %}
```

Comment:
This is clearer, but I am still unclear as to whether `isPreview` should override `omitdebug`.

How is `omitdebug` set? @kelsonic - any concerns?

Reply:
@mpelzsherman it's just a temp feature we need use with: `yarn build --omitdebug`

Reply:
Sorry for pestering, but this will still cause the same issue we have currently. The logic needs to go: `isPreview or (buildtype != 'vagovprod' and omitdebug == false)`, which I know liquid can't do parentheses, so you could do this instead:

```liquid
{% assign showDebug = buildtype != 'vagovprod' and omitdebug == false %}

{% if isPreview or showDebug %}
```

Reply:
If it's `isPreview`, it doesn't matter what `omitdebug` is, we need to show the debug on the preview server.

Reply:
Also, remember `=` is assignment and `==` is for comparison in liquid

Reply:
@kelsonic Updated the operator, but going to go with:
```
{% if isPreview or buildtype != 'vagovprod' %}
  {% if omitdebug == false %}
    {% assign showDebug = true %}
  {% endif %}
{% endif %}
```

Since it isolated the omitdebug flag and will be easier to remove when we're done with it